src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -396,4 +396,13 @@
   <data name="DoNotAddArchiveItemPathToTheTargetFileSystemPathMessage" xml:space="preserve">
     <value>When creating path for '{0} in method {1}' from relative archive item path to extract file and the source is an untrusted zip archive, make sure to sanitize relative archive item path '{2} in method {3}'</value>
   </data>
+  <data name="DoNotUseInsecureRandomness" xml:space="preserve">
+    <value>Do Not Use Insecure Randomness</value>
+  </data>
+  <data name="DoNotUseInsecureRandomnessDescription" xml:space="preserve">
+    <value>Using a cryptographically weak pseudo-random number generator may allow an attacker to predict what sensitive value will be generated.</value>
```

Comment:
>Using a cryptographically weak pseudo-random number generator may allow an attacker to predict what sensitive value will be generated. [](start = 11, length = 134)

Using a cryptographically weak pseudo-random number generator may allow an attacker to predict what security sensitive value will be generated. Use a cryptographically strong random number generator if an unpredictable value is required, or ensure that weak pseudo-random number isn't used in a security sensitive manner.